src/README.md

```diff
@@ -101,3 +101,28 @@ export SECRET_KEY="file:///secret-key-path"
 docker run -it -p 9527:9527 -e REDIRECT_HOST=http://192.168.55.182 -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -e PRIVATE_KEY=$PRIVATE_KEY -e PUBLIC_KEY=$PUBLIC_KEY -v $PWD:/data -v $PWD/secret-key-path:/pulsar-manager/secret-key-path apachepulsar/pulsar-manager:v0.1.0 /bin/sh
 ```
 
+### Enable Github Login
```

Comment:
It is not clear whether every user from github will be able to access the web application.
Can we write explicitly about the real of allowed users?

Reply:
I think this depends on the user who deployed the pulsar-manager platform. If he wants to use a third-party login to authenticate the user, this option is useful to him. This is the first pr that fully supports authentication and authorization https://github.com/apache/pulsar-manager/wiki/Authentication-and-Authorization-in-Pulsar-Manager. After this series of work https://github.com/apache/pulsar-manager/issues/14 is completed, pulsar-manager will have the ability to deeply integrate with pulsar's multi-tenants.